reference/test/matrix/dense_kernels.cpp

```diff
@@ -54,10 +55,15 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/matrix/sparsity_csr.hpp>
 
 
+#include "core/base/unaligned_access.hpp"
+#include "core/matrix/bccoo_aux_structs.hpp"
 #include "core/matrix/dense_kernels.hpp"
 #include "core/test/utils.hpp"
 
 
+#define BCCOO_BLOCK_SIZE_TESTED 1
```

Comment:
Also make `constexpr` or test multiple values in separate tests.

Reply:
Done!!